### DISCUSSION TITLE
Cannot add Linked Issues

### DISCUSSION THREAD

**Discussion by roboquaternion** [community/community#147919] [New to GitHub] [answered]
I am new to GitHub. I am trying to make make many issues on my board and would like to play around with Linked Issues to learn how they work. The [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/adding-sub-issues) says that I can click "Create Linked Issues" from the sidebar of an existing issue. But I cannot find that button. Am I missing something?

Here is a screenshot of an existing issue's sidebar. This was taken on MacOS and Brave browser. I also tried Firefox and Safari, in regular and incognito mode. I have not tried Windows yet.

<img width="303" alt="image" src="https://github.com/user-attachments/assets/e004e1b2-ba31-408f-8b5a-54b71f4ed132" />

**Comment by lemorage** [comment, marked as answer]
watch the note in the [documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/using-issues/adding-sub-issues) here
> $${\color{blue}\textbf{Note}}$$
> Issue types, sub-issues, and advanced issue search are currently in an opt-in public preview for organizations.

**Reply by roboquaternion** [reply]
What a bummer that only organizations can do it so far. I hope they make this available soon for individuals. For now I am using `-[ ]`  in markdown, then convert to an issue. In a restricted sense it works OK.
